### DISCUSSION TITLE
60 Hz Std Type

### DISCUSSION THREAD

**Discussion by jeacueto24** [e2nIEE/pandapower#1710] [Q&A]
Hi, 

I recently found this great program (PandaPower) in my quest to learn about python to use for my work. I want to give thanks to the people behind this incredible project and would like to raise a question regarding the frequency value which gave a notice for the standard type library that they are only compatible with the 50 Hz system. I've read that you can create your std_type but can I create one that can cater to 60 Hz system?

Thank you for your time.

Kind regards,
jeacueto24

**Comment by SteffenMeinecke** [comment]
Dear @jeacueto24,
it is possible to set the system frequency directly in the `pandapowerNet` object.
You can also pass the the system frequency directly when create the net: `pandapower.create_empty_network(f_hz=60)`.
Please reopen this question if you like to discuss this further.
